src/server/game/DungeonFinding/LFG.h

```diff
@@ -51,6 +51,7 @@ enum LfgUpdateType
     LFG_UPDATETYPE_PROPOSAL_FAILED               = 8,
     LFG_UPDATETYPE_PROPOSAL_DECLINED             = 9,
     LFG_UPDATETYPE_GROUP_FOUND                   = 10,
+    LFG_UPDATETYPE_READDED_TO_QUEUE              = 11,
```

Comment:
The client does not list this value explicitly - however sending anything other than 1, 6 and 10 makes it display "You have been readded to front" text